lib/example2uri.dart

```diff
@@ -0,0 +1,10 @@
+class Example2Uri {
+  final String _exampleName;
+
+  Example2Uri(this._exampleName);
+
+  String get path => 'public/docs/_examples/$_exampleName/dart';
```

Comment:
In this case I have a slight preference for a `class` since it nice holds the two related services (getters) together. It also buys us a bit more flexibility: we could easily subclass to create an alternative service (say, for TS example URIs).